scripts/test/lib/cluster.sh

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env bash
 
+source k8s.sh
```

Comment:
I think good to add the entire path, like [here,](https://github.com/aws/amazon-vpc-resource-controller-k8s/blob/b0e626d0795aa6f991c1ae93cae462a077536245/scripts/test/install-vpc-cni.sh#L10) moving this file would break this.

Reply:
Yes, good catch! Will update, thanks.